lib/timber/logger_backend.ex

```diff
@@ -179,6 +181,13 @@ defmodule Timber.LoggerBackend do
   defp output_event(ts, level, message, metadata, state) do
     %{transport: transport, transport_state: transport_state} = state
 
+    module = Keyword.get(metadata, :module)
+    fun = Keyword.get(metadata, :function)
+    file = Keyword.get(metadata, :file)
+    line = Keyword.get(metadata, :file)
+    %RuntimeContext{module: module, function: fun, file: file, line: line}
+    |> Timber.add_context()
```

Comment:
Oh wait, no. I totally misread this. You should not be calling `Timber.add_context/1` here. That does zilch for us, because it just adds the data you've specified to the current process' context dictionary. The event's log line context is in metadata, so you actually want to be retrieving the runtime information inside of `Timber.LogEntry.new/4` and merging it into the context manually around here: https://github.com/timberio/timber-elixir/blob/master/lib/timber/log_entry.ex#L54.